internal/provider/resource_subaccount_role_collection.go

```diff
@@ -117,6 +119,9 @@ func (rs *subaccountRoleCollectionResource) Read(ctx context.Context, req resour
 	state.Name = types.StringValue(cliRes.Name)
 	state.Description = types.StringValue(cliRes.Description)
 
+	// Setting ID of state - required by hashicorps terraform plugin testing framework for Import . See issue https://github.com/hashicorp/terraform-plugin-testing/issues/84
+	state.Id = state.Name
```

Comment:
It's pretty ugly, that we have to fake the id here just for the sake of the framework. However, it would be better if we'd set it to the same value which is required by the import. Which means subaccount id + role collection name:

`6aa64c2f-38c1-49a9-b2e8-cf9fea769b7f,Destination Administrator`

Reply:
Adjusted